rules/collections/android/any.yaml

```diff
@@ -1,43 +1,49 @@
 collections:
   - id: Collections.Android.Form.Email
-    name: Android app email input
+    name: Android Form Email
     patterns: 
       - "(?i)email.*"
     tags:
+      sourceId: Data.Sensitive.ContactData.EmailAddress
 
   - id: Collections.Android.Form.User
-    name: Android app username input
+    name: Android Form User Account
     patterns: 
       - "(?i)user.*"
     tags:
+      sourceId: Data.Sensitive.AccountData.AccountID
 
   - id: Collections.Android.Form.Address
-    name: Address text input Android
+    name: Android Form Address
     patterns: 
       - "(?i)address.*"
     tags:
+      sourceId: Data.Sensitive.ContactData.Address
 
   - id: Collections.Android.Form.PhoneNumber
-    name: Phone number input Android
+    name: Android Form Phone Number
     patterns: 
       - "(?i)phone.*"
     tags:
+      sourceId: Data.Sensitive.ContactData.PhoneNumber
 
   - id: Collections.Android.Form.ZipCode
-    name: Zip code input Android
+    name: Android Form Zip Code
     patterns: 
       - "(?i)zip.*"
     tags:
+      sourceId: Data.Sensitive.ContactData.Address
 
   - id: Collections.Android.Form.Password
-    name: Password input Android
+    name: Android Form Password
     patterns: 
       - "(?i)password.*"
     tags:
+      sourceId: Data.Sensitive.AccountData.AccountPassword
   
   - id: Collections.Android.Form.Sensitve
-    name: Miscellaneous sensitive text input Android
+    name: Android Form Sensitive Data
     patterns: 
       - ".*(?i)(bio|message|detail|comment).*"
     tags:
-
+      sourceId: Data.Sensitive.AccountData.AccountPassword
```

Comment:
One thing to note here is that there is not source Id for generic sensitive data like comments, bio, message. So I just added a placeholder here. Any ideas what we can add here?

Reply:
@HiteshMah-Jan Need your input

Reply:
for message we have Emails or Text Messages

Reply:
bio doesn't seems to be a PII ? sounds something like instagram-bio.